db/migrate/20160329163708_add_carrier_and_service_level_to_spree_shipping_methods.rb

```diff
@@ -0,0 +1,6 @@
+class AddCarrierAndServiceLevelToSpreeShippingMethods < ActiveRecord::Migration
+  def change
+    add_column :spree_shipping_methods, :carrier, :string
```

Comment:
I'm adding a deface template and I was looking at what Glossier did on their side for this and it doesn't appear that they needed permitted attributes.